package.json

```diff
@@ -19,6 +19,7 @@
     "gatsby-transformer-remark": "^1.7.25",
     "gatsby-transformer-sharp": "^1.6.16",
     "gray-matter": "3.1.1",
+    "node-gyp": "^5.0.3",
```

Comment:
@Chrizz2343, what is node-gyp used for and is it necesary for production? if not, I'd take it out.